code/modules/mob/living/simple_animal/guardian/guardian.dm

```diff
@@ -258,14 +258,31 @@
 /mob/living/simple_animal/hostile/guardian/punch
 	melee_damage_lower = 20
 	melee_damage_upper = 20
+	armour_penetration = 75
 	damage_transfer = 0.5
-	playstyle_string = "As a standard type you have no special abilities, but have a high damage resistance and a powerful attack capable of smashing through walls."
+	playstyle_string = "As a standard type you have the ability to stop time for everyone but you and your summoner, but have a high damage resistance and a powerful attack capable of smashing through walls."
 	environment_smash = 2
 	magic_fluff_string = "..And draw the Assistant, faceless and generic, but never to be underestimated."
 	tech_fluff_string = "Boot sequence complete. Standard combat modules loaded. Holoparasite swarm online."
 	bio_fluff_string = "Your scarab swarm stirs to life, ready to tear apart your enemies."
 	var/battlecry = "AT"
 
+/mob/living/simple_animal/hostile/guardian/punch/New()
+	..()
+	var/obj/effect/proc_holder/spell/aoe_turf/conjure/timestop/holoparasite/holoparasite = null
+
+	holoparasite = new /obj/effect/proc_holder/spell/aoe_turf/conjure/timestop/holoparasite
+	holoparasite.clothes_req = 0
+	holoparasite.human_req = 0
+	holoparasite.player_lock = 0
+	AddSpell(holoparasite)
+
+	var/obj/effect/proc_holder/spell/self/timestopimmunity/timestopimmunity = null
+
+	timestopimmunity = new /obj/effect/proc_holder/spell/self/timestopimmunity
+
+	AddSpell(timestopimmunity) // I don't know how the hell it comes that the Holoparasite needs both timestop and the immunity to work, as humans can just have either and still walk around in it.
```

Comment:
You might be able to get around this using the `immune` list defined in the `effect/timestop`:

https://github.com/HippieStationCode/HippieStation13/blob/56a3fdddeb58b6e0dedb20514f4cc43bee628c58/code/modules/research/xenobiology/xenobiology.dm#L549

Here you can see it skips people who are in the list. If you add summoner to this list then when the para uses it both the para and summoner should be immune.

Here's where the caster is added (usually the wizard):

https://github.com/HippieStationCode/HippieStation13/blob/56a3fdddeb58b6e0dedb20514f4cc43bee628c58/code/modules/research/xenobiology/xenobiology.dm#L538